apps/website/src/server/db/show-and-tell.ts

```diff
@@ -498,3 +503,46 @@ export async function getPostsToShow() {
 
   return postsToShow;
 }
+
+// Fetch all mod comments for a specific entry
+export async function getModCommentsForEntry(entryId: string) {
+  return await prisma.showAndTellModComment.findMany({
+    where: { entryId },
+    include: { user: true },
+  });
+}
```

Comment:
Dito used for both admin and public data, fetches *all* mod comments (including internal ones) and selects all fields from the mod user (incl. email)! Public data should not contain any mod user info, admin data only the username.